test/functional/test_pin.py

```diff
@@ -18,6 +21,42 @@ def resources(client):
 	return Resources(client)
 
 
+def path_to_hash(client, path):
+    return path.split("/")[-1]
+
+
+def is_pinned(client, path):
+	error_msg = None
+	try:
+	    resp = client.pin.ls(path)
+	    assert path_to_hash(path) in resp["Keys"]
```

Comment:
There is probably no point in having the `path_to_hash` function separate here.

Reply:
removed